### PR TITLE
Update duplex.md to use the latest string contact.

### PR DESCRIPTION
**Pull request by Anduin2017**
Update duplex.md to use the latest string contact method.

## Summary

Describe your changes here.

Update duplex.md to use the latest string contact method.

### CODE REVIEW

File: docs/framework/wcf/samples/duplex.md [hunk 1]
```diff
@@ -55,14 +55,14 @@ public class CalculatorService : ICalculatorDuplex
   
     public void Clear()  
     {  
-        Callback.Equation(equation + " = " + result.ToString());  
+        Callback.Equation($"{equation} = {result.ToString()}");
```
Comment (rpetrusha) [inline docs/framework/wcf/samples/duplex.md]:
```suggestion
        Callback.Equation($"{equation} = {result}");  
```

File: docs/framework/wcf/samples/duplex.md [hunk 2]
```diff
@@ -55,14 +55,14 @@ public class CalculatorService : ICalculatorDuplex
...
-        Callback.Equation(equation + " = " + result.ToString());  
+        Callback.Equation($"{equation} = {result.ToString()}");  
         equation = result.ToString();  
     }  
   
     public void AddTo(double n)  
     {  
         result += n;  
-        equation += " + " + n.ToString();  
+        equation += $" + {n.ToString()}";
```
Comment (rpetrusha) [inline docs/framework/wcf/samples/duplex.md]:
```suggestion
        equation += $" + {n}";  
```

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thanks, @Anduin2017, for modernizing the code in this example. I've suggested some changes, since the ToString method is called automatically for interpolated expressions that are not of type **string**.

**Comment by Anduin2017** [comment]
@rpetrusha Wow! Thanks for your suggestion! That makes it really simple now!

**Review by EdiWang — APPROVED** [review]

**Comment by rpetrusha** [comment]
Thanks, @Anduin2017, for making the additional changes. This looks good. I'll merge your PR, and your changes should be live on docs.microsoft.com in the next day or two.

**Review by rpetrusha — APPROVED** [review]

**Pull request merged**